src/app/about/page.tsx

```diff
@@ -25,6 +26,22 @@ export default async function About() {
 
   return (
     <Container maxWidth="lg" className="about-page">
+      <Head>
```

Comment:
It seems like the `Head` component from `next/head` might be deprecated?

I notice in the compilation step, it notes:

```
Warning: You're using `next/head` inside the `app` directory, please migrate to the Metadata API. See https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#step-3-migrating-nexthead for more details.
```

I also notice that in the current commit, the JSON is not rendered in the DOM:

![image](https://github.com/CodeForBc/codeforbc-web/assets/16601729/32967c4f-5647-448c-b3c7-4ec51d664b62)

Notice how I search for `ld+json` in the DOM, but no results show up, whereas if I follow the documentation recommendation here, using `section`:

https://nextjs.org/docs/app/building-your-application/optimizing/metadata#json-ld

I get:

![image](https://github.com/CodeForBc/codeforbc-web/assets/16601729/4d080cd9-902c-480e-9859-b3148178485d)

Reply:
addresssed in [79215cc](https://github.com/CodeForBc/codeforbc-web/pull/23/commits/79215cc69494fc1e96bdb055878a7ccae3f7cc43), [f702e5b](https://github.com/CodeForBc/codeforbc-web/pull/23/commits/f702e5bc5aded3ddb4fa522b65004fd1ff76fce5), [317dcc8](https://github.com/CodeForBc/codeforbc-web/pull/23/commits/317dcc88c67319718a9af52ca98c9e38cbaf41df), [959bf14](https://github.com/CodeForBc/codeforbc-web/pull/23/commits/959bf147fcf200a72c09c6bf6a7fdc3a5f3aa43f), [13de453](https://github.com/CodeForBc/codeforbc-web/pull/23/commits/13de45333e59aec4ead6adeed4e3fcd6f8e69aa7)